.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,115 @@
+---
+name: 🚀 Feature Request
+description: |
+  Submit a proposal for a new feature
+# title: "[Feature]"
+labels: ["feature-request", "P3-low"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## ❗Important Announcement
+
+        ### 🚧 Temporary Delay in Feature Requests and Pull Request Reviews
+
+        **At this time, we may be slower to respond to new feature requests and review pull requests. Existing requests and PRs will remain in the backlog but may not be prioritized immediately.**
+
+        - **Reason**: Our current focus is on addressing bugs, improving system performance, and implementing essential updates. This will help stabilize the project and ensure smoother management.
+        - **Impact**: While no new feature requests or pull requests are being outright rejected, there may be significant delays in reviews. We encourage the community to help by reviewing PRs or assisting other users in the meantime.
+        - **What You Can Do**: If you're interested in contributing, reviewing open PRs by following our [Review Guidelines] or offering support to other users is greatly appreciated. All feature requests and PRs will be revisited once the suspension period is lifted.
+
+        We appreciate your patience and understanding as we continue to improve Uptime Kuma.
+
+        [Review Guidelines]: ../REVIEW_GUIDELINES.md
+
+        ### 🚫 Please Avoid Unnecessary Pinging of Maintainers
+
+        **We kindly ask users to refrain from pinging maintainers unless absolutely necessary. Pings should be reserved for critical issues or urgent matters that require immediate attention.**
+
+        - **Why**: To help maintainers focus on high-priority tasks, we kindly request that unnecessary or repeated pings be minimized. This will help ensure quicker responses for matters that truly need attention.
+        - **What This Means**: Non-urgent pings may be ignored or addressed later. Please assess the urgency of your request before pinging a maintainer.
+
+        #### Guidelines for Pinging
+
+        - ✅ **Appropriate**: Critical issues or urgent matters requiring immediate attention.
+        - ❌ **Not Appropriate**: Repeated or non-urgent pings, which may be ignored or addressed later.
+
```

lets reword this to be shorter too

```suggestion
        ### 🚫 Please Avoid Unnecessary Pinging of Maintainers

        **We kindly ask you to refrain from pinging maintainers unless absolutely necessary. Pings are reserved for critical/urgent pull requests that require immediate attention.**

        *Why*: Reserving pings for urgent matters ensures maintainers can prioritize critical tasks effectively.
```